README.rst

```diff
@@ -0,0 +1,323 @@
+
+    uart:~$
+
+Run ``xu config_list`` to see list of available configurations:
+
+.. code-block:: console
+
+    uart:~$ xu config_list
+    rpi_5_domd
+    rpi_5_domu
+
+DomD control
+============
+
+Create DomD by using ``xu create`` with DomD configuration and Xen domainId ``-d 1``.
+Note the `zephyr-xenlib`_ starts automatically domain with Xen domainId ``1``.
+
+.. code-block:: console
+
+    uart:~$ xu create rpi_5_domd -d 1
+    W: Domain device tree generation is not supported
+    I: rambase = 40000000, ramsize = 16777216
+    I: kernbase = 40000000 kernsize = 360452, dtbsize = 11058
+    I: kernsize_aligned = 2097152
+    I: DTB will be placed on addr = 0x40e00000
+    (XEN) memory_map:add: dom1 gfn=107d517 mfn=107d517 nr=1
+    uart:~$ (XEN) d1v0: vGICD: unhandled word write 0x000000ffffffff to ICACTIVER0
+    (XEN) common/grant_table.c:1909:d1v0 Expanding d1 grant table from 1 to 2 frames
+    ...
+    (XEN) common/grant_table.c:1909:d1v0 Expanding d1 grant table from 15 to 16 frames
+
+At this moment RPI5 led should start blinking.
+
+Pause DomD - RPI5 led should stop blinking:
+
+.. code-block:: console
+
+    uart:~$ xu pause 1
+
+Unpause DomD - RPI5 led should start blinking again:
+
+.. code-block:: console
+
+    uart:~$ xu unpause 1
+
+Attach to DomD console with ``xu console``, use ``Ctrl-']'`` to exit console:
+
+.. code-block:: console
+
+    uart:~$ xu console 1
+    Attached to a domain console
+    LED state: OFF
+    [00:00:00.000,000] <inf> xen_events: xen_events_init: events inited
+
+    [00:00:00.000,000] <inf> uart_hvc_xen: Xen HVC inited successfully
+
+    *** Booting Zephyr OS build v3.6.0-72-g702a8af0cbe9 ***
+    LED state: ON
+    LED state: OFF
+
+DomU control
+============
+
+Create DomU by using ``xu create`` with DomU configuration and Xen domainId ``-d 2``.
+
+.. code-block:: console
+
+    uart:~$ xu create rpi_5_domu -d 2
+    W: Domain device tree generation is not supported
+    I: rambase = 40000000, ramsize = 16777216
+    I: kernbase = 40000000 kernsize = 364548, dtbsize = 11058
+    I: kernsize_aligned = 2097152
+    I: DTB will be placed on addr = 0x40e00000
+    I: Created domain is paused
+    To unpause issue: xu unpause 2
+    uart:~$ xu unpause 2
+
+Create DomU by using ``xu create`` with DomU configuration and Xen domainId ``-d 0``.
+Note the `zephyr-xenlib`_ will assign Xen domainId automatically with ``-d 0``.
+
+.. code-block:: console
+
+    uart:~$ xu create rpi_5_domu -d 0
+    W: Domain device tree generation is not supported
+    I: rambase = 40000000, ramsize = 16777216
+    I: kernbase = 40000000 kernsize = 364548, dtbsize = 11058
+    I: kernsize_aligned = 2097152
+    I: DTB will be placed on addr = 0x40e00000
+    I: Created domain is paused
+    To unpause issue: xu unpause 3
+                    ^^^^^^^^^^^^^^ Xen domainId assigned to quest domain
+
+Now unpause DomU quest domains with ``xu unpause``:
+
+.. code-block:: console
+
+    uart:~$ xu unpause 2
```

Comment:
Probably only one of these lines should remain

Reply:
no. example intended to create them as paused.